make/photon/registry/builder

```diff
@@ -1,44 +1,32 @@
 #!/bin/bash
 
-set +e
+set -e
 
 if [ -z $1 ]; then
-  error "Please set the 'version' variable"
+  echo "Please set the 'version' variable"
   exit 1
 fi
 
 VERSION="$1"
 
-set -e
-
 # the temp folder to store binary file...
 mkdir -p binary
 rm -rf binary/registry || true
 
 cd `dirname $0`
-cur=$PWD
 
 # the temp folder to store distribution source code...
 TEMP=`mktemp -d ${TMPDIR-/tmp}/distribution.XXXXXX`
-git clone -b $VERSION https://github.com/distribution/distribution.git $TEMP
+git clone -b $VERSION --depth 1 https://github.com/distribution/distribution.git $TEMP
```

Comment:
why is it necessary to specify the depth here?

Reply:
If `--depth 1` is not specified, then the entire history of all the commits to the `$VERSION` will be cloned, which is unnecessary and is wasteful in terms of transfer amount and processing time (and very painful for a poor environment like mine).
By specifying `--depth 1`, only the commits specified by `$VERSION` can be retrieved.